### ISSUE TITLE
Accessibility violation | aria-label attribute into a generic role 

### ISSUE THREAD

**Issue by thomas-siozos** [Vuepic/vue-datepicker#907]
**Describe the bug**
The input element with **data-test="dp-input"** has an "aria-label" attribute (aria-label="Datepicker input") which is not allowed on a generic role.
That leads to an accessibility violation.

**To Reproduce**
1. Go to the date picker.
2. Run an a11y tool in your browser, I did it with [this one](https://chrome.google.com/webstore/detail/arc-toolkit/chdkkkccnlfncngelccgbgfmjebmkmce/related).

**Screenshots**
You can run the test even in the examples in the documentation.
![image](https://github.com/Vuepic/vue-datepicker/assets/34145662/bbf7996d-e991-491d-b124-30f39a2e738a)
![image](https://github.com/Vuepic/vue-datepicker/assets/34145662/ece0d44c-e2e2-41c8-852d-1ce2cbdacec6)

**Desktop & mobile (please complete the following information):**
 - Browser chrome.
